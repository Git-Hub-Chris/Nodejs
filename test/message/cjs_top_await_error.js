'use strict';

await Promise.resolve();

