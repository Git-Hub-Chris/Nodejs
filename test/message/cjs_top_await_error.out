/*/test/message/cjs_top_await_error.js:3
await Promise.resolve();
^^^^^

SyntaxError: Top-Level await is only supported in ESM.
    at wrapSafe (internal/modules/cjs/loader.js:**:**)
    at Module._compile (internal/modules/cjs/loader.js:**:**)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:**:**)
    at Module.load (internal/modules/cjs/loader.js:**:**)
    at Function.Module._load (internal/modules/cjs/loader.js:**:**)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:**:**)
    at internal/main/run_main_module.js:**:**

